.cirrus.yml

```diff
@@ -119,14 +120,29 @@ report_stagetwo_task:
             PROJECT: ${CENTOS_PROJECT}
             BUILD_NAME: ${CENTOS_NAME}
             VM_IMAGE_NAME: ${CENTOS_IMAGE_NAME}
-        - env:
-            PROJECT: ${SOS_PROJECT}
-            BUILD_NAME: ${FEDORA_NAME}
-            VM_IMAGE_NAME: ${FEDORA_IMAGE_NAME}
-        - env:
-            PROJECT: ${UBUNTU_PROJECT}
-            BUILD_NAME: ${UBUNTU_NAME}
-            VM_IMAGE_NAME: ${UBUNTU_IMAGE_NAME}
+      #  - env:
+      #      PROJECT: ${SOS_PROJECT}
+      #      BUILD_NAME: ${FEDORA_NAME}
+      #      VM_IMAGE_NAME: ${FEDORA_IMAGE_NAME}
+      #  - env:
+      #      PROJECT: ${UBUNTU_PROJECT}
+      #      BUILD_NAME: ${UBUNTU_NAME}
+      #      VM_IMAGE_NAME: ${UBUNTU_IMAGE_NAME}
     remove_sos_script: *remove_sos
     setup_script: *setup
     main_script: PYTHONPATH=tests/ avocado run -t stagetwo tests/
+
+report_foreman_task:
+    skip: "!changesInclude('.cirrus.yml', '**/__init__.py', '**/{foreman,foreman_tests,candlepin,pulp,pulpcore}.py')"
```

Comment:
I would add also `apache.py`, since `foreman` and `pulp*` relies on `httpd` config a lot.